tools/test-backend

```diff
@@ -376,7 +376,7 @@ def main() -> None:
         cov = coverage.Coverage(
             data_suffix="", config_file="tools/coveragerc", concurrency="multiprocessing"
         )
-        # Do not clean .coverage file in CircleCi job so that coverage data can be uploaded.
+        # Do not clean .coverage file in GitHub Actions job so that coverage data can be uploaded.
```

Comment:
I'd just say "in continuous integration job" here since there's no reason to specifically name the platform.